maxim/logger/smolagents/client.py

```
@@ -0,0 +1,539 @@
+    scribe().set_level(get_log_level(debug))
+    
+    try:
+        # Import Smolagents classes
+        from smolagents import MultiStepAgent, CodeAgent, ToolCallingAgent
+        # Try to import ManagedAgent if available
+        try:
+            from smolagents import ManagedAgent
+        except ImportError:
+            ManagedAgent = None
+    except ImportError as exc:
+        raise ImportError("Smolagents library is required for instrumentation") from exc
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)